lib/src/client/transport/http2_credentials.dart

```diff
@@ -21,7 +21,7 @@ import '../../shared/security.dart';
 /// returns `true`, the bad certificate is allowed, and the TLS handshake can
 /// continue. If the handler returns `false`, the TLS handshake fails, and the
 /// connection is aborted.
-typedef bool BadCertificateHandler(X509Certificate certificate, String host);
+typedef bool BadCertificateHandler(X509Certificate certificate, String? host);
```

Comment:
I don't think `host` can be nullable (at least based on my reading of code which calls this callback).

Reply:
Done